xray/audit/java/gradle.go

```diff
@@ -152,22 +152,48 @@ func (dtp *depTreeManager) createDepTreeScript() (tmpDir string, err error) {
 	if err != nil {
 		return
 	}
-	depsRepo := ""
-	releasesRepo := ""
 	if dtp.server != nil {
-		releasesRepo, err = getDepTreeArtifactoryRepository(fmt.Sprintf("%s/%s", dtp.releasesRepo, remoteDepTreePath), dtp.server)
-		if err != nil {
-			return
-		}
-		depsRepo, err = getDepTreeArtifactoryRepository(dtp.depsRepo, dtp.server)
+		dtp.releasesRepo, dtp.depsRepo, err = getRemoteRepos(dtp.releasesRepo, dtp.depsRepo, dtp.server)
 		if err != nil {
 			return
 		}
 	}
-	depTreeInitScript := fmt.Sprintf(depTreeInitScript, releasesRepo, depsRepo)
+	depTreeInitScript := fmt.Sprintf(depTreeInitScript, dtp.releasesRepo, dtp.depsRepo)
 	return tmpDir, errorutils.CheckError(os.WriteFile(filepath.Join(tmpDir, depTreeInitFile), []byte(depTreeInitScript), 0666))
 }
```

Comment:
Let's rename the "createDepTreeScript" function to "createDepTreeScriptAndGetDir".

Reply:
done